aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1589,7 +1626,7 @@ impl VMValidator for AptosVM {
         };
 
         let resolver = &state_view.as_move_resolver();
-        let mut session = self.0.new_session(resolver, SessionId::txn(&txn), true);
+        let mut session = self.0.new_session(resolver, SessionId::txn(&txn), false);
```

Comment:
makes sense to me but let's add a comment mentioning we do txn validation with aggregator disabled (and maybe a rationale). I suppose based on the method comment that here anyway we only run the prologue (so no aggregators currently) and want to make sure those checks happen in a straightforward way.

Reply:
Added a comment.